.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
Please remove everything under each of the headings as it's just extra content that whoever opens an issue will need to remove themselves. Each section should be the heading followed by two empty lines:

```md
**Heading**


**Another heading**
```